### PR TITLE
Initial high score display on main menu.

### PR DESCRIPTION
**Pull request by JohnCom**
Adding a high score display to the main screen which updates based on the highest score(s) achieved in the game. Recorded in local file (single highest scorer).

### CODE REVIEW

**Review by WintonNeg — APPROVED** [review]

**Comment by WintonNeg** [comment]
Going to wait for @manan16007  to approve before merging, but seems good

**Review by manan16007 — APPROVED** [review]
Seems good to me . Happy to have it.

**Pull request merged**
